Sources/Extensions/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -104,6 +104,21 @@ public extension Optional where Wrapped: Collection {
 
 }
 
+// MARK: - Methods (Comparable)
+
+extension Optional: Comparable where Wrapped: Comparable {
+    public static func < (lhs: Optional, rhs: Optional) -> Bool {
+        switch (lhs, rhs) {
+        case (.some(let lhsValue), .some(let rhsValue)):
+            return lhsValue < rhsValue
+        case (_, .none):
```

Comment:
@guykogus What's the criteria to return false and true in both those cases?

Reply:
Do you mean why is it `false` if we have `.some < .none` and `.none < .none`? For me the idea was that nothing will ever be less than `.none`. Turns out that this is how Python 2 implements it, as well.

Reply:
Got it 👍 
Personally, I see the point of why it is useful, only concern is if that is correct considering the Swift perspective. By the way, the point you raised is making me think if it should be a `sort(keypath:)` where the key is optional. It is, in fact, useful but again not sure if it's correct.